README.md

```diff
@@ -1,12 +1,81 @@
+
+**Why Trieste?**  
+- Highly modular design and easily customizable. Extend it with your custom model or acquisition functions. Ideal for practitioners that want to use it in their systems or for researchers wishing to implement their latest ideas.
+- Seamless integration with TensorFlow. Leveraging fully its auto differentiation - no more writing of gradients for your acquisition functions!, and scalability capabilities via its support for highly parallelized modern hardware (e.g. GPUs).
+- General purpose toolbox. Advanced algorithms covering all corners of Bayesian optimization and Active learning - batch, asynchronous, constraints, multi-fidelity, multi-objective - you name it, Trieste has it. 
+- Versatile model support out-of-the-box. From gold-standard Gaussian processes (GPs; [GPflow](https://github.com/GPflow/GPflow)) to alternatives like sparse variational GPs, Deep GPs ([GPflux](https://github.com/secondmind-labs/GPflux)) or Deep Ensembles ([Keras](https://keras.io/)), that scale much better with the number of function evaluations.
+- Real-world oriented. Our Ask-Tell interface allows users to apply Bayesian optimization across a range of non-standard real-world settings where control over black-box function is partial. Built on TensorFlow and with comprehensive testing Trieste is production-ready.
+
+
+## Getting started
+
+Here's a quick overview of the main components of a Bayesian optimization loop. 
+
+Let's set up a synthetic black-box objective function we wish to minimize, for example, a popular Branin optimization function, and generate some initial data
+```python
+from trieste.objectives import Branin, mk_observer
+
+observer = mk_observer(Branin.objective)
+
+initial_query_points = Branin.search_space.sample(5)
+initial_data = observer(initial_query_points)
+```
+
+First step is to crate a probabilistic model, for example a Gaussian Process
+```python
+from trieste.models import build_gpr, GaussianProcessRegression
+
+gpflow_model = build_gpr(initial_data, Branin.search_space)
+model = GaussianProcessRegression(gpflow_model)
+```
+
+Next ingredient is to choose an acquisition rule and acquisition function
+```python
+from trieste.acquisition import EfficientGlobalOptimization, ExpectedImprovement
+
+acquisition_rule = EfficientGlobalOptimization(ExpectedImprovement())
+```
+
+Finally, we optimize the acquisition function using our model for a number of steps and we obtain the obtained minimum
+```python
+from trieste.bayesian_optimizer import BayesianOptimizer
+
+bo = BayesianOptimizer(observer, search_space)
+num_steps = 15
+result = bo.optimize(num_steps, initial_data, model)
+query_point, observation, arg_min_idx = result.try_get_optimal_point()
```

Comment:
since we stress the ask-tell, perhaps we should use it here?

Reply:
Not in the getting started I think. Though you could explicitly link both the expected-improvement and ask-tell tutorials if you want.